README.md

```diff
@@ -21,14 +20,38 @@ sequence of commands:
 
 1. `git clone <where ever we host the skeleton>`
 2. Set environment variable:
-    * `export CONVO_WEBHOOK_URL=<MEDALLIA_CONVERSATION_HOST>`
-    * `export SHARED_SECRET=<32_CHARACTER_STRING>`
-    * `export ACCESS_TOKEN=<STRING that is configured in the channel>`
+    * `export CONVO_WEBHOOK_URL=<MEDALLIA_CONVERSATION_HOST>/cg/mc/custom/<CHANNEL_GUID>`
+      1. For conversations inbound messages
+        * `export AUTH_TYPE_INBOUND=<It can be 'Oauth2' or 'Signature'>`
+        * `export SHARED_SECRET=<32_CHARACTER_STRING only for Signature>` 
```

Comment:
We are fully deprecating Signature.  Only show OAuth2 at this point.  Also, the standard name is `OAuth` for capitalization correctness.

Reply:
is it the same for API-Token in Outbound messages?